package/samplers/smac_sampler/README.md

```diff
@@ -41,6 +62,7 @@ sampler = SMACSampler(
         "y": optuna.distributions.IntDistribution(-10, 10),
     },
     n_trials=n_trials,
+    output_directory="smac3_output",
```

Comment:
```suggestion
    output_directory=Path("smac3_output"),
```

Reply:
Thank you.
Having to pass a `Path` type value is inconvenient, so I changed the API to accept `str` and internally converted it to `Path`.